sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Cast.scala

```diff
@@ -773,7 +773,7 @@ case class Cast(child: Expression, dataType: DataType, timeZoneId: Option[String
          |  ${if (i != 0) s"""$buffer.append(" ");""" else ""}
          |
          |  // Append $i field into the string buffer
-         |  ${ctx.javaType(ft)} $field = ${ctx.getValue(row, ft, s"$i")};
+         |  ${ctx.javaType(ft)} $field = ${CodeGenerator.getValue(row, ft, s"$i")};
```

Comment:
Some methods that are very frequently used (e.g. `ctx.javaType()`) are not refactored yet. This is because this refactoring updates more files.